src/main/java/org/quantumbadger/redreader/account/RedditAccountManager.java

```diff
@@ -83,9 +84,11 @@ public void onCreate(final SQLiteDatabase db) {
 				"CREATE TABLE %s (" +
 						"%s TEXT NOT NULL PRIMARY KEY ON CONFLICT REPLACE," +
 						"%s TEXT," +
+						"%s TEXT," +
```

Comment:
The `onCreate` method has been changed, but not `onUpdate` -- surely this means the app will crash when updating from one version to another, as the table won't have this new field?